Extension/src/common.ts

```diff
@@ -62,58 +62,66 @@ export function getRawPackageJson(): any {
     return rawPackageJson;
 }
 
-export function getRawTasksJson(): Promise<any> {
-    return new Promise<any>((resolve, reject) => {
-        const path: string | undefined = getTasksJsonPath();
+export async function getRawTasksJson(): Promise<any> {
+    const path: string | undefined = getTasksJsonPath();
+    return new Promise<any>(async (resolve, reject) => {
```

Comment:
When a function is marked `async`, you don't explicitly return a `Promise<>`.  You just `return {...}` or `throw new Error(...)`

Reply:
I revert it to the previous version. I made this change while merging with another conflict.